tasks/debian/install.yml

```
@@ -20,9 +20,9 @@
 
 - name: Debian | Add Tailscale Signing Key
   become: true
-  ansible.builtin.apt_key:
+  ansible.builtin.get_url:
     url: "{{ apt_signkey }}"
-    state: present
+    dest: "{{ apt_keyring_path }}"
```

Add a `mode` to resolve the ansible-lint warning

```suggestion
    dest: "{{ apt_keyring_path }}"
    mode: '0644'
```

---

Updated, thanks! Should I rebase the branch on the latest `main`?

---

Please!